<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>186</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>303</width>
    <height>186</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>303</width>
    <height>186</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>279</width>
        <height>56</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>279</width>
        <height>56</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Meiryo</family>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PrograShop</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="loginLine">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>194</horstretch>
          <verstretch>21</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>194</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>194</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loginBut">
        <property name="text">
         <string>Log-In</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>303</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
